app/Http/Requests/CreateCommentRequest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Http\Requests;
+
+class CreateCommentRequest extends Request
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
```

Comment:
But then again, you're using the `Session::flash` in the other spot. Speaking of that I didn't even see where the message is displayed